liiatools/cans_pipeline/spec/CANS_schema.yml

```diff
@@ -0,0 +1,330 @@
+        CH Response to Consequences (Strength): *0_3_can_be_blank
+        CH Commitment to Self-Control (Strength): *0_3_can_be_blank
+        CH Treatment Involvement (Strength): *0_3_can_be_blank
+        Sexual Aggression*: *0_3_not_blank
+        Relationship (Sexual Aggression*): *0_3_can_be_blank
+        Physical Force/Threat (Sexual Aggression*): *0_3_can_be_blank
+        Planning (Sexual Aggression*): *0_3_can_be_blank
+        Age Differential (Sexual Aggression*): *0_3_can_be_blank
+        Type of Sex Act (Sexual Aggression*): *0_3_can_be_blank
+        Response to Accusation (Sexual Aggression*): *0_3_can_be_blank
+        Temporal Consistency (Sexual Aggression*): *0_3_can_be_blank
+        History of Sexually Aggressive Behaviour (Sexual Aggression*): *0_3_can_be_blank
+        Running Away*: *0_3_not_blank
+        Frequency of Running (Running Away*): *0_3_can_be_blank
+        Consistency of Destination (Running Away*): *0_3_can_be_blank
+        Safety of Destination (Running Away*): *0_3_can_be_blank
+        Involvement in Illegal Activities (Running Away*): *0_3_can_be_blank
+        Likelihood of Return on Own (Running Away*): *0_3_can_be_blank
+        Involvement with Others (Running Away*): *0_3_can_be_blank
+        Realistic Expectations (Running Away*): *0_3_can_be_blank
+        Planning (Running Away*): *0_3_can_be_blank
+        Offending/Criminal Behaviour*: *0_3_not_blank
+        History (Offending/Criminal Behaviour*): *0_3_can_be_blank
+        Seriousness (Offending/Criminal Behaviour*): *0_3_can_be_blank
+        Planning (Offending/Criminal Behaviour*): *0_3_can_be_blank
+        Community Safety (Offending/Criminal Behaviour*): *0_3_can_be_blank
+        Peer Influences (Offending/Criminal Behaviour*): *0_3_can_be_blank
+        Parental Criminal Behaviour (Offending/Criminal Behaviour*): *0_3_can_be_blank
+        Environmental Influences (Offending/Criminal Behaviour*): *0_3_can_be_blank
+        Legal Compliance (Offending/Criminal Behaviour*): *0_3_can_be_blank
+        Fire Setting*: *0_3_not_blank
+        History (Fire Setting*): *0_3_can_be_blank
+        Seriousness (Fire Setting*): *0_3_can_be_blank
+        Planning (Fire Setting*): *0_3_can_be_blank
+        Use of Accelerants (Fire Setting*): *0_3_can_be_blank
+        Intention to Harm (Fire Setting*): *0_3_can_be_blank
+        Community Safety (Fire Setting*): *0_3_can_be_blank
+        Response to Accusation (Fire Setting*): *0_3_can_be_blank
+        Remorse (Fire Setting*): *0_3_can_be_blank
+        Likelihood of Future Fire Setting (Fire Setting*): *0_3_can_be_blank
+        Intentional Misbehaviour: *0_3_not_blank
+        Bullying: *0_3_not_blank
+        Victimisation/Exploitation*: *0_3_not_blank
+        Duration of Exploitation (Victimisation/Exploitation*): *0_3_can_be_blank
+        Age of Onset - Exploitation (Victimisation/Exploitation*): *0_3_can_be_blank
```

Comment:
Age of Onset – Exploitation (Victimisation/Exploitation*)

This line uses a different character for the hyphen, might be worth checking with Hannah to get this changed although not essential